src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/WoodcutterAndroid.java

```diff
@@ -48,6 +49,13 @@ protected boolean chopTree(Block b, BlockMenu menu, BlockFace face) {
             List<Block> list = Vein.find(target, MAX_REACH, block -> Tag.LOGS.isTagged(block.getType()));
 
             if (!list.isEmpty()) {
+                MultiBlockBreakEvent event = new MultiBlockBreakEvent(null, target, list);
+                Bukkit.getServer().getPluginManager().callEvent(event);
+
+                if (event.isCancelled()) {
+                    return true;
+                }
```

Comment:
Not a justified event call.

Reply:
Whats is for you a justified event call?
In ``ExplosiveTool`` it will be called like I call it